dist/templates/widget-category-consent/category-consent-widget.html

```diff
@@ -27,6 +27,7 @@
         {{/each}}
     </div>
     <div class="coi-category-consent__decison">
+        <span class="coi-category-consent__select-all" onclick="CookieInformation.submitAllCategories();">SELECT ALL</span>
```

Comment:
Please use `translations.multiaccept` here. It's "select all" for all languages now.